integration_versions.md

```diff
@@ -0,0 +1,52 @@
+# This is a table of supported integration versions generated from gemfiles.
+
+| Integration | Ruby Min |  Ruby Max | JRuby Min | JRuby Max |
```

Comment:
I think approach in this table answers your question on upper bounds:
> What to do with upper bounds
Given we are reporting the effective `Max` we test againts, we should be already respecting that upper bound. I don't think we need to surface that in the final table.